registry/resumable/resumablerequestreader_test.go

```diff
@@ -75,9 +75,10 @@ func TestResumableRequestHeaderTooMuchFailures(t *testing.T) {
 	}
 	defer resreq.Close()
 
-	expectedError := `Get I%27m%20not%20an%20url: unsupported protocol scheme ""`
 	read, err := resreq.Read([]byte{})
-	assert.Check(t, is.Error(err, expectedError))
+	assert.Assert(t, err != nil)
+	assert.ErrorContains(t, err, "unsupported protocol scheme")
```

Comment:
Though maybe we should keep this as a check instead of assert.

Reply:
Ah, yes, I can update to `is.ErrorContains()`

Reply:
updated; PTAL